src/main/java/thePackmaster/cardmodifiers/strikepack/AddStrikeTagModifier.java

```diff
@@ -15,15 +17,16 @@ public void onInitialApplication(AbstractCard card) {
         super.onInitialApplication(card);
         if (!card.hasTag(AbstractCard.CardTags.STRIKE)){
             card.tags.add(AbstractCard.CardTags.STRIKE);
+            card.applyPowers();
         }
     }
 
     @Override
-    public void alterName(AbstractCard card) {
-        modifyName(card.name + " Strike", card);
-        super.alterName(card);
+    public String nameSuffix(String cardName) {
+        return " Strike";
```

Comment:
This should probably be taken from a JSON file and not hardcoded, right? Translating this mod seems like quite a task, but still best to avoid hardcoded strings.

For miscellaneous strings like this I usually put them in the UIStrings file for the pack.